```
Commit b616b06b98:
1: T7 Title does not match regex (^(Issue #\d+: .*|Pull #\d+: .*|(minor|config|infra|doc|spelling|dependency): .*)$): "added checkstyles commit-message ci test"
Error: Process completed with exit code 1.
```

https://github.com/checkstyle/patch-filters/actions/runs/24703915425/job/72252838475?pr=388


all good.

please remove text commit, lets merge this PR.

--- a/.github/workflows/linter.yml
+++ b/.github/workflows/linter.yml
@@ -1,0 +1,23 @@
+name: Commit Message
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  gitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Install gitlint
+        run: pip install gitlint
+
+      - name: Run gitlint
+        run: gitlint --commits ${{ github.event.pull_request.base.sha }}..${{ github.sha }}

--- a/.gitlint
+++ b/.gitlint
@@ -1,0 +1,12 @@
+[general]
+ignore=body-is-missing,subject-case
+
+[title-max-length]
+line-length=200
+
+[title-match-regex]
+regex=^(Issue #\d+: .*|Pull #\d+: .*|(minor|config|infra|doc|spelling|dependency): .*)$
+
+[title-trailing-punctuation]
+
+[title-trailing-whitespace]

